.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: node_js
+node_js:
+  - "4.2"
+before_script:
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
+  - sleep 3 # give xvfb some time to start
+  - npm install -g selenium-standalone
+  - selenium-standalone install
+  - selenium-standalone start > /dev/null 2>&1 &
+  - sleep 3
```

Comment:
If we ever run into issues with this, we can prob use something like a `npm run webdriver-manager-start > webdriver-manager.out &` and then watch the `webdriver-manager.out` content via a `while 1` loop to see if it has the "listening" output